### PR TITLE
Fix a couple typos.

### PR DESCRIPTION
**Pull request by jplevyak**
Signed-off-by: John Plevyak <jplevyak@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -162,7 +164,9 @@ The gRPC service can be used to start a server which is able to perform back-to-
 
 ```bash
 ➜ bazel-bin/nighthawk_service --help
```
Comment (jplevyak) [inline README.md]: I noticed that the -> is used here but not above in the "Test it" and "Build it" sections.   Is that intentional?

Comment (jplevyak) [reply README.md]: @oschaaf

File: README.md [hunk 2]
```diff
@@ -38,7 +38,9 @@ bazel build -c opt //:nighthawk
 
 ```bash
 ➜ bazel-bin/nighthawk_client --help
+```
 
+```
```
Comment (jplevyak) [inline README.md]: Note: when this section was ```bash it resulted in the bash keywords being colorized which seemed odd.  @oschaaf

File: README.md (hunk 1 above)
Comment (oschaaf) [reply README.md]: No the inconsistency is not intentional. I have been manually updating these sections, but actually I consider the source of truth for this to live over at  https://github.com/envoyproxy/nighthawk/blob/master/source/client/options_impl.cc
I wonder if it's possible to somehow automate updating these docs from that.

**Comment by jplevyak** [comment]
I can't merge, so I'll have to let you do the honors.

**Pull request merged**